helix-view/src/editor.rs

```diff
@@ -291,6 +299,9 @@ pub struct Config {
     pub insert_final_newline: bool,
     /// Enables smart tab
     pub smart_tab: Option<SmartTabConfig>,
+    /// labels characters used in jumpmode
+    #[serde(skip_serializing, deserialize_with = "string_as_chars")]
+    pub jump_label_alphabet: Vec<char>,
```

Comment:
We probably want to validate that the chars are unique with no duplicates

Reply:
done, I didn't change the default because I felt there was no good default choice for qwerty its subjective defending on handsize and so forth.